plugins/generic/externalFeed/templates/homepage.tpl

```diff
@@ -0,0 +1,50 @@
+{**
+* plugins/generic/externalFeed/templates/homepage.tpl
+*
+* Copyright (c) 2003-2013 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* Template for external feed content on journal homepage
+*}
+
+<div id="externalFeedsHome">
+
+	{foreach from=$entries item=entry}
+		<h3>{$entry.feedTitle}</h3>

```

Comment:
Simple pie handles sanitization: (cf https://github.com/kaschioudi/ojs/blob/00b283cd82345e10514bb072443c00da4597fb4d/plugins/generic/externalFeed/simplepie/SimplePie.inc.php#L14820). 

Here's the default list of tags which are filtered out (https://github.com/kaschioudi/ojs/blob/00b283cd82345e10514bb072443c00da4597fb4d/plugins/generic/externalFeed/simplepie/SimplePie.inc.php#L711) 

However It's possible to specify additional tags in case that list is not complete.